integrated_channels/integrated_channel/models.py

```diff
@@ -205,6 +206,76 @@ def clean(self):
                 }
             )
 
+    def get_learner_data_audit_model(self):
+        return LearnerDataTransmissionAudit.get_completion_class_by_channel_code(self.channel_code())
+
+    def get_last_learner(self, error, config_id):
+        """
+        Gets every learner audit associated with the configuration and finds the most recent record.
+        If the boolean parameter 'error' is True, then we are only looking for the most recent
+        audit record where the status > 400.
+        """
+        audit_model = self.get_learner_data_audit_model()
+        if error:
+            learner_audits = audit_model.objects.filter(
+                enterprise_customer_uuid=self.enterprise_customer.uuid,
+                plugin_configuration_id=config_id,
+                status__gte=400,
+            )
+        else:
+            learner_audits = LearnerDataTransmissionAudit.objects.filter(
+                enterprise_customer_uuid=self.enterprise_customer.uuid,
+                plugin_configuration_id=config_id,
+            )
+        if learner_audits:
+            return learner_audits.aggregate(max=Max('completed_timestamp')).get('max')
+        else:
+            return None
+
+    def get_last_content(self, error, config_id):
```

Comment:
similarly to the learner version of this method, you dont need to pass a `config_id` here. use the instance `self.id`.

nit: i'd consider tweakin the method name and docstring to be more clear as to what youre returning. also consider making `error` like `include_only_errors` or something